### DISCUSSION TITLE
Should we include manpages in VNM?

### DISCUSSION THREAD

**Discussion by civier** [neurodesk/neurodesk.github.io#135] [Q&A] [answered]
The system has been minimised by removing packages and content that are not required on a system that users do not log into.
At least for me, it will be useful (not sure how much storage it occupies)

**Comment by stebo85** [comment, marked as answer]
Yes, totally possible to add this back in. It would consume 7MB in addition - I am a big fan of the 80/20 rule - do you think this feature profits 80 percent of our users and its worth the additional space? Wouldn't it be ok to open a browser and search for the packages people need :?

What does everyone else think?
